src/for-students-whyjoin/WhyJoin.tsx

```diff
@@ -1,5 +1,5 @@
 import { Container, Flex, Image } from "@mantine/core";
-import "@/forstudents-whyjoin/WhyJoin.css";
+import "@/for-students-whyjoin/WhyJoin.css";
 
```

Comment:
Unable to comment on the container, but we can center the text when the viewport gets small enough. Looks a little odd on mobile view

The h2 and p in the `whyjoin-text` class container